public/styles/spacings.css

```diff
@@ -0,0 +1,8 @@
+:root {
+  --space-xs: 4px;
+  --space-base: 8px;
+  --space-sm: 16px;
+  --space-md: 24px;
+  --space-lg: 40px;
+  --space-xl: 80px;
```

Comment:
I'm on the fence on this one, whether we should use rem or px. I did px because I don't think that spacing should be related to font size, because it's not font-size spacing. But I'm still wondering. WDYT?

Reply:
I like rem and was using rem for spacing usually...
I think, as 16px=1rem it's something we should do - but it needs to be acceptable on everyone..

Reply:
Let's talk about it in the design system channel